code/datums/diseases/advance/symptoms/meme.dm

```diff
@@ -37,14 +37,11 @@
 	var/mob/living/carbon/M = A.affected_mob
 	if(prob(20 * A.stage) && !M.stat && !HAS_TRAIT(M, TRAIT_MINDSHIELD))
 		M.emote(emote)
-		if(A.stage >= 5)
-			for(var/mob/living/carbon/C in oviewers(M, 7))
+		if(A.stage >= 5 && prob(20))
+			for(var/mob/living/carbon/C in oviewers(M, 4))
 				var/obj/item/organ/eyes/eyes = C.getorganslot(ORGAN_SLOT_EYES)
 				if(!eyes || HAS_TRAIT(C, TRAIT_BLIND) || HAS_TRAIT(C, TRAIT_MINDSHIELD) || istype(C.get_item_by_slot(ITEM_SLOT_HEAD), /obj/item/clothing/head/foilhat))
 					continue
-				for(var/datum/disease/advance/D in C.diseases)
-					if(D.GetDiseaseID() == A.GetDiseaseID())
-						C.emote(emote)
 				if(A.transmission >= 14)
```

Comment:
Check if transmission is greater than or equal to 14 before doing the loop, if its not then there is no point looping at all since you can't infect anyone

Reply:
a fair point

sorry for the delay, have been quite busy